modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/controller/AgentDownloadController.java

```diff
@@ -22,94 +22,111 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Comparator;
+import java.util.regex.Pattern;
 import org.apache.commons.compress.archivers.zip.ZipArchiveEntry;
 import org.apache.commons.compress.archivers.zip.ZipArchiveOutputStream;
 import org.apache.commons.compress.archivers.zip.ZipFile;
-import org.apache.ignite.Ignite;
-import org.apache.ignite.console.config.WebConsoleConfiguration;
+import org.apache.ignite.console.dto.Account;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import static org.apache.ignite.internal.util.io.GridFilenameUtils.removeExtension;
+import static org.springframework.http.HttpHeaders.CACHE_CONTROL;
+import static org.springframework.http.HttpHeaders.CONTENT_DISPOSITION;
+import static org.springframework.http.HttpHeaders.EXPIRES;
+import static org.springframework.http.HttpHeaders.PRAGMA;
 
 /**
  * Controller for download Web Agent API.
  */
+@RestController
 public class AgentDownloadController {
     /** Buffer size of 30Mb to handle Web Agent ZIP file manipulations. */
     private static final int BUFFER_SZ = 30 * 1024 * 1024;
 
     /** */
-    private final Path pathToAgentZip;
+    @Value("${agent.folder.name}")
+    private String agentFolderName;
 
     /** */
-    private final String agentFileName;
+    @Value("${agent.file.mask}")
```

Comment:
I think name of variable is incorrect. This is not mask this is regexp. It may confuse guys from QA, DevOps teams.